src/Tizen.Multimedia.Remoting/WebRTC/MediaCameraSource.cs

```diff
@@ -35,6 +35,39 @@ public sealed class MediaCameraSource : MediaSource
         /// <since_tizen> 9 </since_tizen>
         public MediaCameraSource() : base(MediaType.Video) {}
 
+        /// <summary>
+        /// Gets or sets the camera device id.
+        /// </summary>
+        /// <value>A value that specifies the camera device id.</value>
+        /// <exception cref="InvalidOperationException">MediaSource is not attached yet.</exception>
+        /// <exception cref="ObjectDisposedException">The WebRTC has already been disposed.</exception>
+        /// <since_tizen> 10 </since_tizen>
```

Comment:
if the value here is somehow related to Tizen.Multimedia.CameraDevice, then consider adding `<seealso cref="Tizen.Multimedia.CameraDevice"/>`

Reply:
It's not related.